CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ Mark  items as `Added`, `Changed`, `Fixed`, `Modified`, `Removed`, `Untested Fea
 * Added variables(task_variables) as part of import and export.
 * Added Endpoint `/filter/filter-preference ` for saving user's filter preference data
 * Added new table called filter_preferences to handle filter preference of a user
+* Added new table TaskOutcomeConfiguration to store workflow transition rules
```

Comment:
can we keep the naming convention consistent? The immediate line above uses actual database table name and here we are using model name.

Reply:
Done